Vizzuality/trase
# Reset dimensions on change context to enable default

## Description

![image](https://user-images.githubusercontent.com/9701591/95851985-4f5eed80-0d53-11eb-8868-6a4b0c09053e.png)

We were setting the default for the dimensions (seen on the choropleth) when the dimensions were empty but this only happened the first time we accessed the tool. After changing the context we should have the same effect so we reset the dimensions when we finish selection on the tool panel.

We still keep the blank selection if the user deselects all dimensions. 

## Testing instructions

Go to a context then select a different one from the tool node panel. The default choropleth should show.
